src/wp-includes/block-template-utils.php

```diff
@@ -1309,7 +1309,7 @@ function wp_generate_block_templates_export_file() {
 	$theme_json_raw = $tree->get_data();
 	// If a version is defined, add a schema.
 	if ( $theme_json_raw['version'] ) {
-		$theme_json_version = 'wp/' . substr( $wp_version, 0, 3 );
+		$theme_json_version = 'wp/' . substr( $version, 0, strpos( $version, '-' ) );
```

Comment:
I think this is just a typo, but version is undefined.

Reply:
```suggestion
		$theme_json_version = 'wp/' . substr( $wp_version, 0, strpos( $wp_version, '-' ) );
```

Reply:
Thanks